ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/verticaltabs/v1/verticaltabs/verticaltabs.html

```diff
@@ -64,6 +64,7 @@
          data-sly-test="${(wcmmode.edit || wcmmode.preview)}"></sly>
 </div>
 <sly data-sly-use.clientlib="core/wcm/components/commons/v1/templates/clientlib.html">
- <sly data-sly-test="${wcmmode.edit}" data-sly-call="${clientlib.js @ categories='core.forms.components.verticaltabs.v1.contentframe'}"/>
+    <sly data-sly-test="${wcmmode.edit}" data-sly-call="${clientlib.js @ categories='core.forms.components.verticaltabs.v1.contentframe'}"/>
+    <sly data-sly-test="${wcmmode.edit}" data-sly-call="${clientlib.css @ categories='core.forms.components.verticaltabs.v1.contentframe'}"/>
```

Comment:
You can also use clientlib.all to load css+js from the categories through a single sightly stmnt.

Reply:
Was looking for syntax to load both. Thanks !!